Packages/com.unity.inputsystem/InputSystem/Editor/InputActionAsset/CopyPasteUtility.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using UnityEditor;
+
+namespace UnityEngine.Experimental.Input.Editor
+{
+    class CopyPasteUtility
+    {
+        const string m_InputAssetMarker = "INPUTASSET\n";
```

Comment:
Naming of internal consts is kConstant.

Reply:
Fixed. Isn't the official standard using "k_" tho?